### PR TITLE
Basic curses rendering setup.

### PR DESCRIPTION
**Pull request by alexhsamuel**

### CODE REVIEW

**Comment by alexhsamuel** [comment]
@KyleSJohnston what's our code review policy?

I suggest (at least cursory) review by two people for each commit. :)

**Comment by KyleSJohnston** [comment]
That policy works for me. (I've set it up such that we need at least one person with write access to approve already.)

File: tbl/screen.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
+from   __future__ import absolute_import, division, print_function, unicode_literals
```
Comment (KyleSJohnston) [inline tbl/screen.py]: Are you using a 2.7 interpreter?

**Review by KyleSJohnston — APPROVED** [review]
Looks good to me.

File: tbl/screen.py (hunk 1 above)
Comment (alexhsamuel) [reply tbl/screen.py]: I put this in before we decided to ditch Python 2. :)

**Pull request merged**